### ISSUE TITLE
Preferences window needs some love

### ISSUE THREAD

**Issue by gergles** [avaidyam/Parrot#101]
The slider at the top of the general tab is unlabeled, as are the two button groups below it.

The last option to show recents and conversations together is too long for the size of the window and so has an ellipsis at the end.

The text and accounts tabs both have no options. However, if you click on one, then go back to the general tab, the spacing is wrong because the preferences window resizes when you select one of the other tabs.

Before clicking a different tab:
<img width="400" alt="screen shot 2018-03-27" src="https://user-images.githubusercontent.com/2148340/38001555-68f1a8b8-31fb-11e8-928e-c6edf4e2b932.png">

After:
<img width="400" alt="screen shot 2018-03-27 2" src="https://user-images.githubusercontent.com/2148340/38001577-8965b3aa-31fb-11e8-9067-88c167e60a04.png">

**Comment by avaidyam** [comment]
Good catch - not sure how I missed this. The Text and Accounts did used to have stuff in them, but I'm trying to migrate the whole preferences code first before migrating the UI from the storyboard it used to be in. Also, multi-account support is going to be a little rough, so it may not happen until v1.0+ (unless I get help on that!)
